src/tests/ftest/soak/soak.py

```diff
@@ -517,18 +719,27 @@ def run_soak(self, test_param):
         """
         self.soak_results = {}
         self.pool = []
+        self.harasser_joblist = []
+        self.harasser_results = {}
         self.test_timeout = self.params.get("test_timeout", test_param)
         self.job_timeout = self.params.get("job_timeout", test_param)
+        self.harasser_timeout = self.params.get("harasser_timeout", test_param)
         self.test_name = self.params.get("name", test_param)
         self.nodesperjob = self.params.get("nodesperjob", test_param)
         self.test_iteration = self.params.get("iteration", test_param)
         self.task_list = self.params.get("taskspernode", test_param + "*")
-
+        self.h_list = self.params.get("harasserlist", test_param + "*")
         job_list = self.params.get("joblist", test_param + "*")
         pool_list = self.params.get("poollist", test_param + "*")
         rank = self.params.get("rank", "/run/container_reserved/*")
-        obj_class = self.params.get(
-            "object_class", "/run/container_reserved/*")
+
+        if self.h_list is not None and "rebuild" in self.h_list:
```

Comment:
This line looks like it's doing what is being done in the `is_harasser()` method. Here you could do:
`if self.is_harasser("rebuild"):` similar to how you are doing it in line 341.

Reply:
yes... make sense...  I have a function I should use it :)

Reply:
Please review changes  thanks!